tests/test_mlbase.py

```diff
@@ -0,0 +1,74 @@
+import lightgbm as lgb
```

Comment:
Please add the tests under `tests/frameworks` - either in the already existing `test_mlbase.py` file or in a new `test_lgbm.py` file